src/components/WeeklyForecast.svelte

```diff
@@ -0,0 +1,153 @@
+<script>
+  import weatherService from '../services/weatherService';
+  import {
+    Table,
+    TableBody,
+    TableBodyCell,
+    TableBodyRow,
+    TableHead,
+    TableHeadCell,
+  } from 'flowbite-svelte';
+  import moment from 'moment';
+  import { link } from 'svelte-spa-router';
+  import weatherCodes from '../assets/weather-interpretation-code-description.json';
+  import { t } from 'svelte-i18n';
+
+  export let latitude;
+  export let longitude;
+
+  const weatherData = {
+    isLoading: false,
+    hasError: false,
+  };
+  let forecastData = [];
+  const routeBase = '/details';
```

Comment:
`routeBase` sounds like route ti `/`, lets rename it to `detailsRoute` or smth like that